packages/integration-sdk-runtime/src/synchronization/index.ts

```diff
@@ -29,6 +29,9 @@ export { createEventPublishingQueue } from './events';
 const UPLOAD_BATCH_SIZE = 250;
 const UPLOAD_CONCURRENCY = 6;
 
+// Let's max at 5 meg even though 6 will technically work
+const UPLOAD_SIZE_MAX = 5000000;
```

Comment:
Fair point.  I don't have any solid reason for that much wiggle room.  I think all we'd really need space for is header information.  I guess my main thought was that even a 5 MB upload size was pretty generous.  But I take your point about the cost of finding the extra 1 MB of data to remove.

Reply:
I updated the max size to 6 MB - 16 KB.  I believe that most servers will reject a header larger than 16 KB, so that should be more than plenty of space for it.